krypy/deflation.py

```diff
@@ -625,12 +625,11 @@ def _auto():
 
         # perturbation
         # HACK until numpy.linal.svd (and thus numpy.linalg.norm) is fixed
-        from scipy.linalg import svd
```

Comment:
We're still using `svd` in the next line... what's the reason for removing it? I guess what we originally wanted is the 2-norm of a matrix, so if that "hack" is not required anymore and we can just use `numpy.linalg.norm` then that's fine of course. I just don't see how it can run without the import now.

Reply:
`svd` is already imported a few lines earlier. https://github.com/andrenarchy/krypy/blob/9cc533b5b2502b85ab4821a7d2e73341c743766a/krypy/deflation.py#L587

(Side note: I'm not sure what the `HACK` comment there refers to; did we file a bug report on that at the time? We might be able to ditch it.)

Reply:
ohh I see. ok, that's fine then of course. I added an issue for the `HACK` comment: https://github.com/andrenarchy/krypy/issues/67